src/components/base-layout/base-layout.tsx

```diff
@@ -1,62 +1,57 @@
-import { ReactNode, Suspense } from "react";
-import {
-  AppBar,
-  Container,
-  ContainerProps,
-  IconButton,
-  Typography,
-} from "@mui/material";
-import { Link as RouterLink, useMatch } from "react-router-dom";
-import { theme } from "../../lib";
-import { Home } from "@mui/icons-material";
+import {ReactNode, Suspense} from "react";
+import {AppBar, Container, ContainerProps, IconButton, Typography,} from "@mui/material";
+import {Link as RouterLink, useMatch} from "react-router-dom";
+import {theme} from "../../lib";
+import {Home} from "@mui/icons-material";
 
 type BaseLayoutProps = {
-  title: string;
-  children: ReactNode;
-  outletProps?: ContainerProps;
+    title: string;
+    children: ReactNode;
+    outletProps?: ContainerProps;
+    maxWidth?: "sm" | "s" | "m";
 };
 
-function BaseLayout({ title, children, outletProps = {} }: BaseLayoutProps) {
-  const { sx, ...outletPropsRest } = outletProps;
-  const isAnwendungsauswahl = useMatch("/");
+function BaseLayout({title, children, outletProps = {}, maxWidth = "sm"}: BaseLayoutProps) {
```

Comment:
Do we still need the `maxWidth` prop? Or can we use the `outletProps` instead?

Reply:
yes, youre right, at this point i didnt understood wherefore outletProps are and how they are working.

I removed maxWidth and moved this config to the outletProps. 

Maybe there is a better way as configure the same outletProps-Config on each page ..?